src/util.ml

```diff
@@ -595,23 +595,23 @@ value parent_has_title conf base p =
         - Vrai si : magicien ou ami ou la personne est public
```

Comment:
I dont understand the question/issue. This file had zilions of mis-coded accented characters.
Recoded the whole file in utf-8. Is this the problem?